org.scala-ide.sdt.core/src/org/scalaide/ui/internal/editor/decorators/bynameparams/CallByNameParamAtCreationPresenter.scala

```diff
@@ -48,14 +48,36 @@ object CallByNameParamAtCreationPresenter extends HasLogger {
           super.traverse(tree)
         }
 
+        def isSynthetic(tree: compiler.Tree) = {
+          Option(tree.symbol).exists(_.isSynthetic)
+        }
+
+        def isByNameParam(param: compiler.Symbol) = {
+          param.isByNameParam || referencesByNameParam(param)
+        }
+
+        /*
+         * This should cover partially applied functions referencing by-name-params (see #1002381).
+         */
+        def referencesByNameParam(param: compiler.Symbol) = {
+          if (!param.hasRawInfo) {
+            false
+          } else {
+            param.rawInfo.typeSymbol match {
```

Comment:
@dragos There is no specific reason for that... the tests also pass if you replace `rawInfo` with `info`, which should return the same result anyway if I read the sources correctly. Should I create another pull request for that, or do you just want to change that yourself?